docs/cookbook.rst

```diff
@@ -50,15 +57,93 @@ adding a ``crispy`` template filter to the form. For example:
     <form action="{% url 'envelope-contact' %}" method="post">
         {% csrf_token %}
         {% antispam_fields %}
-        {{ form|crispy }}
-        <button type="submit">Send!</button>
+        {{ crispy form }}
     </form>
 
 .. _`Bootstrap`: http://getbootstrap.com/
 .. _`django-crispy-forms`: https://github.com/maraujop/django-crispy-forms
 
+To add a submit button, create a custom form using ``django-crispy-forms`` helper:
+
+# forms.py
```

Comment:
Please fix the indentation in the filename comments to match the following code, otherwise code highlighting gets confused :(